testing/tests/DevExpress.ui.widgets/pager.tests.js

```diff
@@ -584,6 +584,28 @@ function() {
         assert.equal(instance._pages[1].value(), 2, 'second page value');
     });
 
+    // T966318
+    QUnit.test('Pager displays duplicated page numbers', function(assert) {
+        const $pager = $('#container').dxPager({
```

Comment:
It seems that the test should be named: "Pager does not display duplicated page numbers"